### PR TITLE
docs(webhooks): Add documentation for custom trust store

### PR DESCRIPTION
**Pull request by ezimanyi**
Documentation for the solution to https://github.com/spinnaker/spinnaker/issues/3189.

### CODE REVIEW

File: reference/pipeline/stages/index.md [hunk 1]
```diff
@@ -235,6 +235,9 @@ custom webhook stage is a webhook stage specifically named and configured for
 your application's needs, which shows up in the standard pipeline stages
 dropdown menu.
 
+It is possible to add additional certification authorities to trust when
+making webhook calls over HTTPS by following these [instructions](/guides/operator/webhook-custom-trust-store/).
```
Comment (dorbin) [inline reference/pipeline/stages/index.md]: "You can [add more certification authorities]() to trust when making webhook calls over HTTPS."

File: guides/operator/webhook-custom-trust-store.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+will not be used for any other connection initiated by Spinnaker. There is no
+way to specify additional CAs on a per-webhook basis; the additional CAs will apply to
+all webhooks.
+
+## Creating a trust store
+
+Create a trust store in Java KeyStore (JKS) format via:
+```bash
+keytool -import -file <path-to-ca-certificate> -alias <name-of-first-ca> -keystore <name-for-keystore>.jks
+
```
Comment (lwander) [inline guides/operator/webhook-custom-trust-store.md]: Extra newline?

File: guides/operator/webhook-custom-trust-store.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+CA. Webhooks to these servers over `https://` will fail using the default configuration.
+
+In order to support this latter use case, Spinnaker allows users to supply additional
+CAs to trust in addition to the default ones. These additional CAs will be used when
+negotiating connections for outbound webhooks (including preconfigured webhooks) but
+will not be used for any other connection initiated by Spinnaker. There is no
+way to specify additional CAs on a per-webhook basis; the additional CAs will apply to
+all webhooks.
+
+## Creating a trust store
```
Comment (dorbin) [inline guides/operator/webhook-custom-trust-store.md]: s/Creating/Create

File: guides/operator/webhook-custom-trust-store.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+You will be prompted to create a password for the new key store, which you'll need to supply to
+Spinnaker in the next step.
+
+After creating the key store with the above command, you can add additional CAs to the keystore
+by running the same command but supplying a different CA certificate and alias. You'll be prompted
+for the keystore password before the new CA can be added. As this trust store will augment the default
+trust store, there is no need to add all of the root CAs to this custom trust store; only CAs that are
+not in the default trust store need to be added.
+
+## Configuring Spinnaker to use the trust store
```
Comment (dorbin) [inline guides/operator/webhook-custom-trust-store.md]: s/Configuring/Configure/

File: guides/operator/webhook-custom-trust-store.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+The trust store can be configured via Halyard as follows:
+```bash
```
Comment (lwander) [inline guides/operator/webhook-custom-trust-store.md:51]: I would remind the user that they'll be prompted here as well for their password

File: guides/operator/webhook-custom-trust-store.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+PEM format, `<name-of-first-ca>` is an arbitrary alias for that CA, and `<name-for-keystore>` is
+the name of a keystore that will be created.
+
+You will be prompted to create a password for the new key store, which you'll need to supply to
+Spinnaker in the next step.
+
+After creating the key store with the above command, you can add additional CAs to the keystore
+by running the same command but supplying a different CA certificate and alias. You'll be prompted
+for the keystore password before the new CA can be added. As this trust store will augment the default
+trust store, there is no need to add all of the root CAs to this custom trust store; only CAs that are
```
Comment (dorbin) [inline guides/operator/webhook-custom-trust-store.md]: s/there is no/you don't/

File: guides/operator/webhook-custom-trust-store.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+not in the default trust store need to be added.
+
+## Configuring Spinnaker to use the trust store
+
+The trust store can be configured via Halyard as follows:
```
Comment (dorbin) [inline guides/operator/webhook-custom-trust-store.md]: You can just delete this sentence

**Review by dorbin — APPROVED** [review]
Just a few nits. 
Thanks for the doc, Eric!

**Review by dorbin — APPROVED** [review]
:+1:

**Pull request merged**